### ISSUE TITLE
`struct light_info` is unused

### ISSUE THREAD

**Issue by lrineau** [CGAL/cgal#176]
@maxGimeno: In the branch cgal-public-dev/CGAL-Qt5_support-GF, the following files define a `struct light_info` but it is never used.
```
Polyhedron/demo/Polyhedron/Scene_edit_polyhedron_item.cpp
Polyhedron/demo/Polyhedron/Scene_polyhedron_selection_item.cpp
Polyhedron/demo/Polyhedron/Scene_nef_polyhedron_item.cpp
Polyhedron/demo/Polyhedron/Scene_polygon_soup_item.cpp
Polyhedron/demo/Polyhedron/Scene_polyhedron_item.cpp
Polyhedron/demo/Polyhedron/Scene_polylines_item.cpp
Polyhedron/demo/Polyhedron/Polyhedron_demo_mesh_3_plugin_cgal_code.cpp
Polyhedron/demo/Polyhedron/Scene_textured_polyhedron_item.cpp
Polyhedron/demo/Polyhedron/Scene_points_with_normal_item.cpp
```
What is more, the file `Polyhedron/demo/Polyhedron/Scene_nef_polyhedron_item.cpp` have a big function that is commented: `Scene_nef_polyhedron_item::uniform_attrib`. It should be removed.

**Comment by lrineau** [comment]
I still see the unused `light_info` struct in the following three files:
```
Polyhedron/demo/Polyhedron/Scene_edit_polyhedron_item.cpp
Polyhedron/demo/Polyhedron/Scene_nef_polyhedron_item.cpp
Polyhedron/demo/Polyhedron/Polyhedron_demo_mesh_3_plugin_cgal_code.cpp
```

**Comment by maxGimeno** [comment]
Done

**Comment by lrineau** [comment]
Fixed in a6ea2d94b75af7f0dd16a9adbbd5cb44fd19b2d2.

Next time, Maxime, add a text similar to
> Fix #176

in your commit message. That closes the bug automatically, with a reference to the commit.
